rs-permission-converter/test/permconv.test.js

```diff
@@ -0,0 +1,67 @@
+const os = require('os');
+const path = require('path');
+const fs = require('fs');
+const fsExtra = require('fs-extra');
+const exec = require('@actions/exec');
+
+// Used by @actions/tool-cache. Directory must exist before we load module.
+const outputDir = path.join(__dirname, '..', 'test_output_dir');
+
+if (!process.env.NEXUS_USERNAME) {
+  // eslint-disable-next-line no-console
+  console.log('Using local user credentials in test');
+  process.env.NEXUS_USERNAME = process.env.GITHUB_USER;
+  process.env.NEXUS_PASSWORD = process.env.GITHUB_TOKEN;
+}
+
+// Directory must exist for tool-cache.
+fsExtra.mkdirs(outputDir);
+
+// We must import pkgbuilder AFTER preparing the environment.
+const { convertPermissions, downloadPermConvTool } = require('../src/permconv');
+
+jest.setTimeout(30000);
+
+describe('RS Permission Converter Tests', () => {
+  beforeAll(() => {
+    fsExtra.mkdirs(outputDir);
+  });
+
+  afterAll(() => {
+    fsExtra.removeSync(outputDir);
+  });
+
+  test('It downloads the build tool', async () => {
+    const tool = await downloadPermConvTool({
+      binaryVersion: '1.0.0',
+    });
+    expect(fs.existsSync(tool)).toEqual(true);
+  });
+
+  // There's no binary we can test on MacOS
+  if (os.platform() !== 'darwin') {
+    test('the permission converter is called correctly for sql', async () => {
+      exec.exec = jest.fn();
```

Comment:
This will break exec.exec for any tests that runs after this test. Use `jest.mock('@actions/exec')` to mock the module before it is first referenced.